helm/sealed-secrets/templates/ingress.yaml

```diff
@@ -13,6 +13,9 @@ metadata:
     {{- if .Values.ingress.annotations }}
     {{- include "sealed-secrets.render" ( dict "value" .Values.ingress.annotations "context" $) | nindent 4 }}
     {{- end }}
+    {{- if .Values.commonAnnotations }}
+    {{- include "sealed-secrets.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+    {{- end }}
```

Comment:
This is inside a `.Values.ingress.annotations` if clause, I think it should be out of it.

Reply:
Now removed the first clause for consistency with the other manifests, as the `.Values.ingress.annotation` was checked 2 times.